### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by tiserett**
[DEMO LINK](https://tiserett.github.io/react_people-table-basics/)

### CODE REVIEW

File: src/components/Loader/PeopleTable.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people?.map(person => (<PersonLink person={person} people={people}/>))}
```
Comment (Githi54) [inline src/components/Loader/PeopleTable.tsx:28]:
```suggestion
        {people?.map(person => (
        <PersonLink 
              person={person} 
              people={people}
         />))}
```

**Review by Githi54 — APPROVED** [review]
Well done!🦾

File: src/components/PersonLink.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+  return (
+    <tr
+      data-cy="person"
+      className={(`/people/${slug}` === location && 'has-background-warning') || ''}
+    >
+      <td>
+        <NavLink
+          to={`/people/${slug}`}
+          className={sex === 'f' ? 'has-text-danger' : ''}
```
Comment (AlinaMatuschak) [inline src/components/PersonLink.tsx:34]: use classnames

File: src/pages/PeoplePage.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import {
+  NoPeopleMessage,
+  PeopleLoadingError,
+} from '../components/ErrorMessage';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+import { Person } from '../types';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
```
Comment (AlinaMatuschak) [inline src/pages/PeoplePage.tsx:12]:
```suggestion
  const [people, setPeople] = useState<Person[]>([]);
```

**Review by AlinaMatuschak — APPROVED** [review]
👍

**Pull request closed without merging**